models/provisioning.cattle.io.cluster.js

```diff
@@ -103,7 +103,7 @@ export default class ProvCluster extends SteveModel {
       action:     'rotateEncryptionKey',
       label:      this.$rootGetters['i18n/t']('nav.rotateEncryptionKeys'),
       icon:       'icon icon-refresh',
-      enabled:     this.isRke1 && this.mgmt?.hasAction('rotateEncryptionKey') && this.mgmt?.isReady,
+      enabled:     (this.isRke2 && this.mgmt?.isReady && this.canUpdate) || (this.isRke1 && this.mgmt?.hasAction('rotateEncryptionKey') && this.mgmt?.isReady),
```

Comment:
Could we split this in 2 clear statements? Also we double check `this.mgmt?.isReady`, so maybe we define it once? Not sure if we want to underline the need to be ready.

```ts
const enabledRKE2 = this.isRke2 && this.mgmt?.isReady && this.canUpdate;
const enabledRKE1 = this.isRke1 && this.mgmt?.hasAction('rotateEncryptionKey') && this.mgmt?.isReady;
```

Reply:
Those two are not quite the same thing because the first one is checking if you have the ability to edit RKE2 clusters in general, whereas the RKE1 one is checking if the Norman API is allowing the specific action for rotating encryption keys.

I have cleaned up the action list as well as I can and added the comment

```
Actions are not supported in the Steve API, so we check
available actions for RKE1 clusters, but not RKE2 or K3s clusters.
```